src/cnaas_nms/api/app.py

```diff
@@ -121,7 +126,7 @@ def handle_error(self, e):
 
 # SocketIO on connect
 @socketio.on('connect')
-@jwt_required()
+@jwt_required
```

Comment:
I added a config option 'OAUTH2_ENABLED' in the ApiSettings class. In 'tools/security' I use that setting to override the decorator. When 'OAUTH2_ENABLED' is True it will execute the decorator before the function as you describe above.